opm/parser/eclipse/Python/Python.hpp

```diff
@@ -70,11 +70,11 @@ class EclipseState;
   True     |  OFF               |  *                  | { }
```

Comment:
I do not understand what the third column means: is it "Have we already constructed an instance?"

What does the * mean?

For TRY, I do not understand the Result column. Assuming the third column meaning as above, when we already have constructed an instance, we suddenly get no instance? I would think that you would refer to the existing one? That is, assuming `{}` means "no Python for you", and `{ Python }` means "yes, Python can be used", and the `if (python)` check in the example above returns `{false|true}` respectively.